packages/nextjs/pages/example-zk/SignedStats.tsx

```diff
@@ -0,0 +1,128 @@
+import type { NextPage } from "next";
+import { useBirthYearProofsStore } from "~~/services/store/birth-year-proofs";
+
+function shortenHashString(hash: string) {
+  const prefix = hash.substring(0, 2);
```

Comment:
```suggestion
  const prefix = hash.substring(0, 4);
```
So that it's recognizable in both ends.